### DISCUSSION TITLE
I want to use kepler container metrics to visualise per week data over a period of month in grafana using promql

### DISCUSSION THREAD

**Discussion by MahimaK20** [sustainable-computing-io/kepler#1038] [Q&A]
Hello All,
I am trying to achieve visualisation of kepler container metrics as per week data graph shown below with prometheus and grafana.
![image](https://github.com/sustainable-computing-io/kepler/assets/150212811/27f6a3f2-7de0-46db-96df-ab502ba171ca)

But I am getting aggregated view of 4 weeks as below. The query I have used is **sum((sum_over_time((irate(kepler_container_dram_joules_total[1w]))[4w:1w])) + (sum_over_time((irate(kepler_container_package_joules_total[1w]))[4w:1w])))**
![image](https://github.com/sustainable-computing-io/kepler/assets/150212811/2c859231-7dd5-44dc-97f2-742d1ccde6ac)

Can you please help me with the solution to achieve my objective.

**Comment by rootfs** [comment]
@sthaha any suggestions?
